scripts/car_models.sql

```diff
@@ -9,4 +10,10 @@ CREATE TABLE IF NOT EXISTS car_models
  model_title character varying(125) NOT NULL,
  year integer NOT NULL,
  PRIMARY KEY (id)
-)
+);
+
+CREATE INDEX first_index
+ON car_models (make_code, model_code);
+
+CREATE INDEX second_index
+ON car_models (make_title, model_title);
```

Comment:
So you should only make an index on make_code because your where clauses only really uses that one. You could also create a index on make_code and model_code since one of your where clauses uses it but it doesn't give you that much of a faster result. Year does not see the same benefits because for each year, there's still a lot of data, so even after sorting them by year, you'd still have to do a lot of searching.